charts/radar-integration/values.yaml

```diff
@@ -166,8 +166,8 @@ networkpolicy:
 oauth_client_id: radar_redcap_integrator
 # -- OAuth2 client secret
 oauth_client_secret: secret
-# -- URL of the Management Portal
-managementportal_url: http://management-portal:8080/managementportal
+# -- URL of the Management Portal. Make sure to include the trailing slash
+managementportal_url: http://management-portal:8080/managementportal/
```

Comment:
Should I update `radar-integration` to just include the slash in the request itself?

Reply:
yes ideally we should use functions to concat parts of the url so less chance of these kind of error (then would work with and without the `/`)